src/index.css

```diff
@@ -33,8 +33,11 @@
   --formal-accent-semi-transparent: rgba(239, 236, 214, 0.4);
   --error-semi-transparent: rgba(255, 123, 131, 0.2);
   --selection: #ff7bed35;
+  --transparent: #00000000;
+}
+* {
+  -webkit-tap-highlight-color: var(--transparent);
```

Comment:
Webkit browser had this `blue-box-shadow` when tapping on any clickable element, maybe removing it not really user-friendly, but with `menu` it certainly looks better, I think we should handle this tap color manually for elements